docs/dev/libs/cep.md

```diff
@@ -168,33 +256,34 @@ start.where(
 </div>
 </div>
 
-<span class="label label-danger">Attention</span> The call to `Context.getEventsForPattern(...)` has to find the 
-elements that belong to the pattern. The cost of this operation can vary, so when implementing your condition, try 
-to minimize the times the method is called.
+<span class="label label-danger">Attention</span> The call to `context.getEventsForPattern(...)` finds all the 
+previously accepted events for a given potential match. The cost of this operation can vary, so when implementing 
+your condition, try to minimize the times the method is called.
 
-**Simple Conditions:** This type of conditions extend the aforementioned `IterativeCondition` class. They are simple 
-filtering conditions that decide to accept an element or not, based only on properties of the element itself.
+**Simple Conditions:** This type of conditions extend the aforementioned `IterativeCondition` class and decides 
```

Comment:
... This type of condition extends the aforementioned `IterativeCondition` class and decides whether ...